server/src/controllers/order.controller.ts

```diff
@@ -32,9 +94,39 @@ const OrderController = {
 
   async getOrdersByUserId(req: Request, res: Response): Promise<Response> {
     try {
-      const { user_id } = req.params;
-      const data = await Order.find({ user: user_id });
-
+      const { user_id } = req.body.user._id;
```

Comment:
This shouldn't be destructured, `user._id` is a string itself.

Reply:
Right, thanks.